hotsos/defs/scenarios/juju/juju_binary_cve.yaml

```diff
@@ -3,14 +3,14 @@ checks:
     binary:
       handler: hotsos.core.plugins.juju.JujuBinaryInterface
       juju:
-        - min: '1.0.0'
-          max: '2.9.49'
+        - min: '2.9.0'
+          max: '2.9.48'
```

Comment:
So, I think there's a bug in the version checker. While I was creating the tests, I tried this (2.9.48) with the juju version 2.9.48, but it wasn't catching, so I bumped it to 49.

Reply:
I run the tests again and for some reason, they're working the way they should, so we are good to go.